src/start_server.ts

```diff
@@ -143,6 +152,23 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxyRoot && options.proxyTarget) {
+    const apiProxy = httpProxy.createProxyServer();
+    const proxyHandler = (req: express.Request, res: express.Response) => {
+      // Remove proxied part from the url
+      req.url = req.url.slice(options.proxyRoot.length + 1);
```

Comment:
If I understand [the documentation](http://expressjs.com/en/api.html#req.originalUrl) correctly, yes. Please correct me if I am wrong